src/combineTests.ts

```diff
@@ -101,7 +101,7 @@ export function combineTests(options:
     testMethods.push(
       shortenClassNames({
         imports : [...existingImports, ...imports, ...staticImports],
-        test : test.test,
+        test : test.test || test.testBody || '',
```

Comment:
This will break stuff elsewhere, but shouldn't it be:

```suggestion
        body: test.body || test.test || test.testBody || '',
```